akka-javasdk/src/main/scala/akka/javasdk/impl/workflow/ReflectiveWorkflowRouter.scala

```diff
@@ -9,17 +9,19 @@ import akka.javasdk.impl.AnySupport
 import akka.javasdk.impl.CommandHandler
 import akka.javasdk.impl.CommandSerialization
 import akka.javasdk.impl.InvocationContext
+import akka.javasdk.impl.serialization.JsonSerializer
 import akka.javasdk.workflow.CommandContext
 import akka.javasdk.workflow.Workflow
-import com.google.protobuf.any.{ Any => ScalaPbAny }
+import akka.runtime.sdk.spi.BytesPayload
 
 /**
  * INTERNAL API
  */
 @InternalApi
 class ReflectiveWorkflowRouter[S, W <: Workflow[S]](
-    override protected val workflow: W,
-    commandHandlers: Map[String, CommandHandler])
+    override val workflow: W,
+    commandHandlers: Map[String, CommandHandler],
+    serializer: JsonSerializer)
     extends WorkflowRouter[S, W](workflow) {
```

Comment:
I don't think we have the need for both ReflectiveWorkflowRouter and WorkflowRouter. Remove WorkflowRouter? I did corresponding for ESE.